lib/Normalize_extended_ast.ml

```diff
@@ -182,15 +187,13 @@ let diff_cmts (conf : Conf.t) x y =
             in
             let len = String.length str - chars_removed in
             let source = String.sub ~pos:1 ~len str in
-            match
-              Parse_with_comments.parse Parse.ast Structure conf ~source
-            with
-            | exception _ -> norm_non_code z
```

Comment:
@julow we used to call `norm_non_code` on the source before, now we don't, but there is no change on the tests. And since this makes the check "stronger" maybe it's a good idea, was it intentional or not?

Reply:
This is when a block of code fails to parse. We format these exactly as they were in the input, and with Odoc's indentation corrections we don't mess the indentation either.
The stronger check works :)

Reply:
Noted, thanks!